workers/src/main/scala/workers/ItcServer.scala

```diff
@@ -36,7 +36,7 @@ object ItcServer extends WorkerServer[IO, ItcMessage.Request] with ItcPicklers {
 
   private def createClient[F[_]: Async]: Client[F] =
     FetchClientBuilder[F]
-      .withRequestTimeout(5.seconds)
+      .withRequestTimeout(10.seconds)
```

Comment:
is this because of the time to upload a sed?

Reply:
I haven't profiled how much it is because of the time ITC needs to pull the custom SED out of S3 and how much is due to ITC maybe needing more time to process a custom SED, if that's the case; but I was getting timeouts with 5 seconds.